src/SFA.DAS.ApplyService.Web/Views/Application/Pages/Index.cshtml

```diff
@@ -50,11 +50,19 @@
                 </div>
             }
 
-            @if (!string.IsNullOrEmpty(@Model.Title) && Model.Questions.Count() > 1)
+            @if (Model.Questions.Count() > 1)
             {
-                <h1 class="govuk-heading-xl">
-                    @Html.Raw(Model.Title)
-                </h1>
+                @if (!string.IsNullOrEmpty(@Model.Title))
+                {
+                    <h1 class="govuk-heading-xl">
+                        @Html.Raw(Model.Title)
+                    </h1>
+                }
+
+                @if (!string.IsNullOrEmpty(@Model.BodyText))
+                {
+                    @Html.Raw(Model.BodyText)
```

Comment:
should be in a div or something

Reply:
@gdbarnes knows what he's doing. ;)

Reply:
Or, more helpfully, the body text itself contains surrounding tags.  Hence the Html.Raw. 👍

Reply:
Yeah.. I know what I'm doing. 100%. No question about it. Don't even ask! 😐